src/config.ts

```diff
@@ -225,35 +225,33 @@ export function getDefaultConfig(): IConfig {
 }
 
 /**
- * Grabs an explicit path provided for mjolnir's config from an arguments vector if provided, otherwise returns undefined.
- * @param argv An arguments vector sourced from `process.argv`.
- * @returns The path if one was provided or undefined.
+ * @returns The users's raw config, deep copied over the `defaultConfig`.
```

Comment:
Was this supposed to be deleted? not sure if the diff is just being weird here though

Reply:
Diff being weird I think, those "removed" lines are from the old "get config path" function

Reply:
oki